modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -1010,6 +1010,105 @@ definition {
 			displayDate = "${displayDate}");
 	}
 
+	@description = "LPS-115305 - Workflow creator user cannot see previous notifications after the task has been assigned to another user"
```

Comment:
Hi @jaydawu ! 
We use the `description` field to describe the scenario to be tested, instead of the ticket summary.
In this case it could be: `Verify that the asset creator can view the workflow notifications after the task is assigned to another user.`
Does it makes sense to you?

Reply:
Sure, thank you!